docs/docs/configuration.md

```diff
@@ -26,114 +26,117 @@ Iceberg tables support table properties to configure table behavior, like the de
 
 ### Read properties
 
-| Property                          | Default            | Description                                            |
-| --------------------------------- | ------------------ | ------------------------------------------------------ |
-| read.split.target-size            | 134217728 (128 MB) | Target size when combining data input splits           |
-| read.split.metadata-target-size   | 33554432 (32 MB)   | Target size when combining metadata input splits       |
-| read.split.planning-lookback      | 10                 | Number of bins to consider when combining input splits |
-| read.split.open-file-cost         | 4194304 (4 MB)     | The estimated cost to open a file, used as a minimum weight when combining splits. |
-| read.parquet.vectorization.enabled| true               | Controls whether Parquet vectorized reads are used     |
-| read.parquet.vectorization.batch-size| 5000            | The batch size for parquet vectorized reads            |
-| read.orc.vectorization.enabled    | false              | Controls whether orc vectorized reads are used         |
-| read.orc.vectorization.batch-size | 5000               | The batch size for orc vectorized reads                |
+| Property                              | Default            | Description                                                                                     |
+|---------------------------------------|--------------------|-------------------------------------------------------------------------------------------------|
```

Comment:
Can we avoid modifying other parts?

Reply:
Yes, I reverted back the markdown re-formating